main.py

```diff
@@ -2,20 +2,47 @@
 import os
 import json
 
-KEY = os.environ.get('TOMORROW_KEY')
-LOCATION = '60448 US'
-TIMESTEPS = '1d'
-UNITS = 'imperial'
+# KEY = os.environ.get('TOMORROW_KEY')\
+# LOCATION = '60448 US'
+# TIMESTEPS = '1d'
+# UNITS = 'imperial'
 
-url = f"https://api.tomorrow.io/v4/weather/forecast?location={LOCATION}&timesteps={TIMESTEPS}&units={UNITS}&apikey={KEY}"
+# url = f"https://api.tomorrow.io/v4/weather/
+#     forecast?location={LOCATION}&timesteps={TIMESTEPS}&units={UNITS}&apikey={KEY}"
 
-headers = {"accept": "application/json"}
+# headers = {"accept": "application/json"}
 
-response = requests.get(url, headers=headers)
-response_data = json.loads(response.text)
+# response = requests.get(url, headers=headers)
+# response_data = json.loads(response.text)
+
+# formatted_response = json.dumps(response_data, indent=4)
+# print(formatted_response)
+
+
+def prompt():
+    location = input(
+        'Would you like to input a zip code or a city name? (Z/C) ')
+    if location == 'Z':
+        return readZip(location)
```

Comment:
do we need this parameter for the readZip and readCity functions? Because if the user inputs "Z", then function will only take in Z and not the actual zip code which isn't really useful.